project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.module

```diff
@@ -170,21 +233,191 @@ function c4m_message_load__c4m_welcome_to_group(Message $message) {
 }
 
 /**
- * Resolves member full name toke.
+ * Resolves addressee user id.
+ *
+ * @param Message $message
```

Comment:
@anvmn 
can you please elaborate why do we need this one and where do we use it ?